google-cloud-tools-plugin/resources/generation/src/appengine/flexible/java/app.yaml

```diff
@@ -0,0 +1,4 @@
+runtime: java
+env: flex
+env_variables:
+  'DBG_ENABLE': 'true'
```

Comment:
after offline discussion it seems that it _should_ work without it, but we'll leave it until we can verify.